package-lock.json

```diff
@@ -0,0 +1,7431 @@
+{
```

Comment:
do we need this file?

Reply:
Yes. It's advisable to always commit lock file to the source control.